mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/VirtualMachineWorkerServiceLocalImpl.java

```diff
@@ -145,38 +140,27 @@ public void onNext(List<Boolean> booleans) {
     @Override
     public void start() {
         logger.info("Starting VirtualMachineWorkerServiceLocalImpl");
-        Schedulers.newThread().createWorker().schedule(new Action0() {
-            @Override
-            public void call() {
-                try {
-                    WrappedExecuteStageRequest request = null;
-                    request = createExecuteStageRequest();
-                    setupRequestFailureHandler(request.getRequest().getTimeoutToReportStart(), request.getRequestSubject(),
-                            new Action0() {
-                                @Override
-                                public void call() {
-                                    logger.error("launch error");
-                                }
-                            });
-                    logger.info("onNext'ing WrappedExecuteStageRequest: {}", request.toString());
-                    executeStageRequestObserver.onNext(request);
-                } catch (MalformedURLException e) {
-                    e.printStackTrace();
-                }
+        Schedulers.newThread().createWorker().schedule(() -> {
+            try {
+                WrappedExecuteStageRequest request;
```

Comment:
merge the two lines

Reply:
If you mean to merge them like that
`Schedulers.newThread().createWorker().schedule(() -> { try { `
It looks more well-formated to me with the line break.